common/djangoapps/track/middleware.py

```diff
@@ -70,7 +83,12 @@ def process_request(self, request):
 
             views.server_track(request, request.META['PATH_INFO'], event)
         except:
-            pass
+            ## Why do we have the overly broad except?
+            ##
+            ## Adding instrumentation. If we're dropping events on the floor,
+            ## we should know about it.
+            event = {'event-type': 'exception', 'exception': sys.exc_info()[0]}
+            views.server_track(request, request.META['PATH_INFO'], event)
 
```

Comment:
We should also consider what happens here if it cannot save the exception event for some reason. We should ensure that doesn't result in a user facing error either.